VCEntities/VCEntities/presentation/presentationExchange/request/PinDescriptor.swift

```diff
@@ -10,4 +10,18 @@ public struct PinDescriptor: Codable, Equatable {
     public let salt: String?
     public let iterations: Int?
     public let alg: String?
+    
+    public init(type: String?,
```

Comment:
why do we need the constructor only now?

Reply:
an internal init is created automatically in structs and is not needed, but I need it to be public for Wallet Library things so making it public now.